tests/vm/basevm.py

```diff
@@ -309,7 +309,7 @@ def console_sshd_config(self, prompt):
     def print_step(self, text):
         sys.stderr.write("### %s ...\n" % text)
 
-    def wait_ssh(self, seconds=300):
+    def wait_ssh(self, seconds=600):
```

Comment:
Next commit overwrites this line anyway.
I do not think that this one is needed

Reply:
After discussing we decided to keep this as a separate commit to make it more clear we fixed this.  Although the line changes, the increase of the timeout stays the same.